.github/workflows/release.yml

```diff
@@ -380,3 +380,80 @@ jobs:
           repository: fermyon/homebrew-tap
           event-type: spin-release
           client-payload: '{"version": "${{ github.ref_name }}"}'
+
+  docker:
+    runs-on: "ubuntu-20.04"
+    needs: [build-and-sign, build-spin-static]
+    permissions:
+      contents: read
+      packages: write
+    env:
+      REGISTRY: ghcr.io
+      IMAGE_NAME: ${{ github.repository }}
+    strategy:
+      matrix:
+        config:
+          - { dockerfile: "Dockerfile", tag-suffix: "" }
+          - { dockerfile: "distroless.Dockerfile", tag-suffix: "-distroless" }
```

Comment:
Hi @vdice, just a curious question: does `distroless` means `static` implicitly?

Reply:
@rajatjindal Yes; well both.  It uses a distroless base image and includes the static Spin binary.  Open to alternative names... let me know.

Reply:
I was wondering if it is worth adding static to the name, if distroless implies static by default, then we can omit it. But otherwise it would be nice to call it out explicitly.

Reply:
Indeed, I do think "distroless implies static by default".